setup.py

```diff
@@ -134,25 +134,14 @@ def run_setup():
       ext_modules = ext_modules,
       entry_points="""
         [console_scripts]
-        results_schema=pyomo.scripting.commands:results_schema
-        pyro_mip_server = pyomo.scripting.pyro_mip_server:main
-        test.pyomo = pyomo.scripting.runtests:runPyomoTests
         pyomo = pyomo.scripting.pyomo_main:main_console_script
+        test.pyomo = pyomo.scripting.runtests:runPyomoTests
+        pyomo_python = pyomo.scripting.commands:pyomo_python
+        pyro_mip_server = pyomo.scripting.pyro_mip_server:main
         pyomo_ns = pyomo.scripting.commands:pyomo_ns
         pyomo_nsc = pyomo.scripting.commands:pyomo_nsc
         kill_pyro_mip_servers = pyomo.scripting.commands:kill_pyro_mip_servers
         launch_pyro_mip_servers = pyomo.scripting.commands:launch_pyro_mip_servers
-        readsol = pyomo.scripting.commands:readsol
-        OSSolverService = pyomo.scripting.commands:OSSolverService
-        pyomo_python = pyomo.scripting.commands:pyomo_python
-        pyomo_old=pyomo.scripting.pyomo_command:main
-
-        [pyomo.command]
-        pyomo.help = pyomo.scripting.driver_help
-        pyomo.test.pyomo = pyomo.scripting.runtests
-        pyomo.pyro_mip_server = pyomo.scripting.pyro_mip_server
-        pyomo.results_schema=pyomo.scripting.commands
-        pyomo.viewer=pyomo.contrib.viewer.pyomo_viewer
```

Comment:
I think we need to leave these.  The pyomo.command entrypoints are all loaded by the main pyomo script (in `pyomo/scripting/pyomo_main.py`).  This is nominally to provide a registration mechanism outside of `pyomo.environ`.  I believe that it even allows other packages to register subcommands for the `pyomo` command.  We may even make use of it, e.g., for PySP to register subcommands.

Reply:
Thanks for clearing that up. I could not find any documentation about subsections in entry points. I'll take a closer look at each of these.